src/System.Private.CoreLib/shared/System/Runtime/CompilerServices/FixedAddressValueTypeAttribute.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.CompilerServices
+{
+    [Serializable]
+    [AttributeUsage(AttributeTargets.Field)]
+    public sealed class FixedAddressValueTypeAttribute : Attribute
+    {
+        public FixedAddressValueTypeAttribute() { }
```

Comment:
This needs a tracking issue to cover feeding information about this attribute to the code generator. This attribute affects GS cookie generation.